lib/Runtime/Types/PathTypeHandler.cpp

```diff
@@ -879,7 +879,8 @@ namespace Js
         }
         if (!(attr & ObjectSlotAttr_Configurable))
         {
-            JavascriptError::ThrowCantDelete(PropertyOperation_None, scriptContext, scriptContext->GetPropertyName(propertyId)->GetBuffer());
+            JavascriptError::ThrowCantDeleteIfStrictModeOrNonconfigurable(
+                flags, scriptContext, scriptContext->GetPropertyName(propertyId)->GetBuffer());
```

Comment:
>flags [](start = 16, length = 5)

Fix for #4118 (to throw when deleting non-configurable properties in strict mode)